connman/unit/test-service.c

```diff
@@ -268,9 +268,9 @@ int connman_provider_get_index(struct connman_provider *provider)
 	return provider ? provider->index : -1;
 }
 
-int connman_provider_get_family(struct connman_provider *provider)
+bool connman_provider_get_family(struct connman_provider *provider, int family)
 {
-	return provider ? provider->family : PF_UNSPEC;
+	return provider && provider->family == family ? true : false;
```

Comment:
nit: that "? true : false" part is superfuous

Reply:
1ecd690ab4681cb5b7964339064d387b561eab38 cleanup